requirements.txt

```diff
@@ -10,4 +10,4 @@ packaging
 # NOTE: resolvelib 0.x version bumps should be considered major/breaking
 # NOTE: and we should update the upper cap with care, at least until 1.0
 # NOTE: Ref: https://github.com/sarugaku/resolvelib/issues/69
-resolvelib >= 0.5.3, < 0.6.0  # dependency resolver used by ansible-galaxy
+resolvelib >= 0.5.3, < 0.9.0  # dependency resolver used by ansible-galaxy
```

Comment:
We might want a note here to update the versions listed in the tests when updating this range.

Reply:
Good idea.